### PR TITLE
More updates, closing issues

### LINKED ISSUE #1652: Fix FreeRCT
If anyone has free time, the following things should be fixed for FreeRCT:
* there is a new homepage at: https://freerct.net
* FreeRCT should be also listed under _RollerCoaster Tycoon 2_, not just _RollerCoaster Tycoon_, because it remakes both games
* its development is pretty active now
* the project had its first release, so the status should probably be _semi-playable_

Thanks.

### LINKED ISSUE #1656: Add OpenKrush
The current KKND you have is no longer being developed. Development moved to this repo: https://github.com/IceReaper/OpenKrush

### LINKED ISSUE #1657: Add CorrinoEngine
Open source reimplementation of Emperor Battle for Dune: https://github.com/cookgreen/CorrinoEngine

### PR DESCRIPTION
**Pull request by hansejo**
- Updating URL in FreeRCT
- Adding Corrino Engine and the original game

Major reorganizing to KKnD:
- Moved to o.yaml
- KKnD becomes OpenKrush

closes #1652
closes #1657
closes #1656

### CODE REVIEW

File: games/c.yaml [hunk 1]
```diff
@@ -904,6 +904,20 @@
...
 
+- name: CorrinoEngine
+  development: active
+  lang:
+  - C#
+  license:
+  - GPL3
+  content: commercial
+  originals:
+  - 'Emperor: Battle for Dune'
```
Comment (cxong) [inline games/c.yaml:915]: add `status`

File: games/o.yaml [hunk 2]
```diff
@@ -1313,6 +1313,38 @@
...
+  status: semi-playable
+  lang:
+  - C++ 
+  license:
+  - GPL3
+  content: free
+  info: Based on original executable decompilation
+  updated: 2022-04-18
+
+- name: OpenKrush
```
Comment (cxong) [inline games/o.yaml:1331]: add status

File: games/o.yaml [hunk 3]
```diff
@@ -1313,6 +1313,38 @@
   video:
     youtube: 5CXqWdbWrVs
 
+- name: OpenKKnD
```
Comment (cxong) [inline games/o.yaml:1316]: please add images and/or a video

Comment (hansejo) [reply games/o.yaml:1316]: It seems like OpenKKnD is a de-compilation project. No screenshots or videos found.
Different story for for OpenKrush however. I've added some.

File: games/o.yaml [hunk 4]
```diff
@@ -1313,6 +1313,43 @@
...
+  framework:
+  - OpenRA
+  content: free
+  info: OpenRA mod
+  updated: 2022-04-26
+  url: https://www.kknd-game.com/about
+  images:
+  - https://media.moddb.com/images/mods/1/38/37924/OpenRA-2018-12-16T105029073Z.png
+  video:
+    youtube: https://www.youtube.com/watch?v=Mz7ZT3lo-MM
```
Comment (cxong) [inline games/o.yaml:1351]:
```suggestion
    youtube: Mz7ZT3lo-MM
```

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/o.yaml (hunk 4 above)
Comment (hansejo) [reply games/o.yaml]: Done

File: games/o.yaml [hunk 5]
```diff
@@ -1313,6 +1313,43 @@
...
+  framework:
+  - OpenRA
+  content: free
+  info: OpenRA mod
+  updated: 2022-04-26
+  url: https://www.kknd-game.com/about
+  images:
+  - https://media.moddb.com/images/mods/1/38/37924/OpenRA-2018-12-16T105029073Z.png
+  video:
+    youtube: Mz7ZT3lo-MM
```
Comment (cxong) [inline games/o.yaml:1351]: this video cannot be embedded

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/o.yaml (hunk 5 above)
Comment (hansejo) [reply games/o.yaml]: Sorry, forgot that YouTube can prevent that. Fixed now.

**Pull request merged**